src/Samples/Sample.AspNetCore.SystemTests/Test/PaymentTests/Payment/PaymentSaleTests.cs

```diff
@@ -30,7 +30,7 @@ public async Task Payment_Swish_Sale(Product[] products, PayexInfo payexInfo)
                 .Actions.Rows[y => y.Name.Value.Contains(PaymentResourceOperations.ViewPayment)].Should.BeVisible()
                 .Actions.Rows.Count.Should.Equal(3);
 
-            var swishPayment = await SwedbankPayClient.Payment.GetSwishPayment(paymentLink, PaymentExpand.All);
+            var swishPayment = await SwedbankPayClient.GetSwishPayment(paymentLink, PaymentExpand.All);
```

Comment:
Is it possible to have this read like the following?

```suggestion
            var swishPayment = await SwedbankPayClient.SwishPayments.Get(paymentLink, PaymentExpand.All);
```

Reply:
I've ripped out that part of the `SwedbankPayClient`. If you want to use one singular part of the api at a time you could just use `IPaymentsClient` instead of the `ISwedbankPayClient`.
But I can see what I can do.